src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,95 @@
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  color: $main-accent;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 16px;
+  transition-duration: 300ms;
+
+  &__image-block {
+    margin: auto;
+  }
+
+  &__image {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 16px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 40px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: end;
+    margin-top: 16px;
+  }
+
+  &__reviews-count {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price-block {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-text {
+    margin: 0;
+    color: $secondary;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price {
+    margin: 0;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__button {
+    margin-top: 16px;
+    height: 40px;
+    border: none;
+    border-radius: 5px;
+    background-color: $blue-accent;
+    color: $white;
+    text-transform: uppercase;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+
+    &:hover {
+      border: 1px solid $blue-accent;
+      background-color: $white;
+      color: $blue-accent;
+      transform: scale(120%);
+    }
```

Comment:
It's good practice to avoid transforming the scale of interactive elements on hover as it can cause layout shifts and affect the user experience negatively. Consider using other ways to indicate hover, like changing the color or adding a shadow.